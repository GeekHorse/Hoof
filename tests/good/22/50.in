new right foo oof done
new down literal literal literal done rab done
new down baz zab done
most up
dig literal literal literal done rab done
value
save
