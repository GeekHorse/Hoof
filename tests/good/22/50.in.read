hello 
new 
ok 
new 
ok 
new 
ok 
ok 
ok 
ok   literal done rab 
ok 
