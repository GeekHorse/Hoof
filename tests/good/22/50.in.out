new right foo oof done
new down literal literal literal done rab done
new down baz zab done
